builtin/credential/cert/path_certs.go

```diff
@@ -114,6 +114,15 @@ formatted as "oid:value". Expects the extension value to be some type of ASN1 en
 All values much match. Supports globbing on "value".`,
 			},
 
+			"metadata_extensions": {
+				Type: framework.TypeCommaStringSlice,
+				Description: `A comma-separated string or array of extensions
+formatted as "oid" or "oid:alias". Upon successfull authentication, these extensions will
```

Comment:
Couple points here.
1. Do we need the 2 forms, oid and oid:alias ? As in, if we just have a comma separate list of oids, and always use the santiized ( dots to dashes ) version of it as the metadata key, would there be an issue ? I'm guessing redabale metadata keys is the argument there, but wanted to check.
2. If we do want the oid:alias form, I'd recommend using a different word and not `alias`, as that is a loaded word, being a key concept in the identity system.

Reply:
We don't need aliases apart from readability wherever the metadata is used, and actually putting the dashes-separated oid as metadata key will make it clear throughout that the metadata originated as certificate extensions. As we're renaming the configuration value to `allowed_metadata_extensions`, it also makes more sense to just accept a list of oids. I'll remove the aliasing.